README.md

```diff
@@ -8,6 +8,12 @@
 
 [![Autotest Status](http://autotest.ardupilot.org/autotest-badge.svg)](http://autotest.ardupilot.org/)
 
+Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
+been under development since 2010 by a team of diverse professional engineers and computer scientists. It is
+the only autopilot software capable of controlling any vehicle system imaginable, from conventional
```

Comment:
i can imagine vehicles we don't support, there is a really cool combined submarine plane that we can't support. I think we need to tone this down in the place this came from too

Reply:
add word 'almost' ?